xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="pit_database_recovery">
  <title>Point-in-Time &mariadb; Database Recovery</title>
  <para>
-  In this scenario, everything is still running (&lcm;, cloud
-  controller nodes, and compute nodes) but you want to restore the &mariadb;
-  database to a previous state.
+  In this scenario, everything is still running (&lcm;, cloud controller nodes,
+  and compute nodes) but you want to restore the &mariadb; database to a
+  previous state.
  </para>
  <section>
   <title>Restore from a Swift backup</title>
-  <orderedlist>
-   <listitem>
+  <procedure>
+   <step>
     <para>
      Log in to the &lcm;.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Determine which node is the first host member in the
      <literal>FND-MDB</literal> group, which will be the first node hosting the
      &mariadb; service in your cloud. You can do this by using these commands:
     </para>
 <screen>cd ~/scratch/ansible/next/ardana/ansible
-grep -A1 FND-MDB--first-member hosts/verb_hosts</screen>
+grep -A1 FND-MDB--first-member hosts/verb_hosts
+[FND-MDB--first-member:children]
+ardana002-cp1-c1-m1</screen>
+   </step>
+   <step xml:id="mariadb-nodes">
     <para>
-     Example output:
+     Find the host IP address which will be used to log in.
     </para>
-<screen>$ grep -A1 FND-MDB--first-member hosts/verb_hosts
-[FND-MDB--first-member:children]
-ardana-cp1-c1-m1</screen>
-   </listitem>
-   <listitem>
+<screen>cat /etc/hosts | grep ardana002-cp1-c1-m1
+10.84.43.82      ardana002-cp1-c1-m1-extapi ardana002-cp1-c1-m1-extapi
+192.168.24.21    ardana002-cp1-c1-m1-mgmt ardana002-cp1-c1-m1-mgmt
+10.1.2.1         ardana002-cp1-c1-m1-guest ardana002-cp1-c1-m1-guest
+10.84.65.3       ardana002-cp1-c1-m1-EXTERNAL-VM ardana002-cp1-c1-m1-external-vm
+    </screen>
     <para>
-     Log in to the first host member from the <literal>FND-MDB</literal> group.
+     In this example, <literal>192.168.24.21</literal> is the IP address for
+     the host.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Become root:
+     SSH into the host.
     </para>
-<screen>sudo su</screen>
-   </listitem>
-   <listitem>
+<screen>ssh ardana@192.168.24.21</screen>
+   </step>
+   <step>
     <para>
-     Source the backup environment file
+     Source the backup file.
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
```

Comment:
Find the <literal>Client ID</literal> for the host name from the beginning of
          this procedure ( <literal>ardana002-cp1-c1-m1</literal> in this example.